ui/pages/question_page.rb

```diff
@@ -0,0 +1,92 @@
+module Pages
+  class QuestionPage < Base
+    set_url '/modules{/module_name}/questionnaires{/question}'
+
+    element :next_button, '.govuk-button', text: 'Next'
+    element :finish_test, '.govuk-button', text: 'Finish test'
+    elements :radio_buttons, '.govuk-radios__label'
```

Comment:
Is it possible to leverage an `#options` id so the `QuestionPage` can be multiple choice agnostic?

Reply:
possibly, this is just a carryover from joseph's branch - still needs a lot of work.